examples/nginx-nodeport.yaml

```diff
@@ -23,6 +23,38 @@ spec:
         volumeMounts:
         - name: listener
           mountPath: /usr/share/nginx/html/listener
+        - name: tls
+          mountPath: /tls
+        - name: listener2
+          mountPath: /usr/share/nginx/html/listener2
+      volumes:
```

Comment:
I can't repro this, are you sure you were running on the https://github.com/stackabletech/secret-operator/pull/310 branch?

Reply:
Yes, I had the correct secret-operator version deployed.

I used stackablectl to create the kind cluster. It uses the cluster name `stackable-data-platform` and the nodes are then named `stackable-data-platform-control-plane` and `stackable-data-platform-worker`. Therefore, I just removed the label `secrets.stackable.tech/node: kind-control-plane` from the Secret. The Listeners have the service type `NodePort`, so the Pod is considered "node scoped" and the Secret requires the node label, otherwise the provisioning of the Secret volume fails (see https://github.com/stackabletech/secret-operator/blob/3abbb69427d30ead0a7979ebbb3097ca85daf60f/rust/operator-binary/src/csi_server/controller.rs#L184-L187).

Reply:
Ah. Did it work when you changed the node label then?

Reply:
Yes, it did.

Reply:
8a3899ece18bdd323c198f2dd4b79e9881fbdd62

Reply:
Isn't this a valid use case which should work?

Reply:
It is! But it's also hard to test without binding the example to the name of the nodes.. :/